meson.build

```diff
@@ -231,7 +231,7 @@ if libarchive.found()
 endif
 endif
 libjcat = dependency('jcat', version: '>= 0.1.4', fallback: ['libjcat', 'libjcat_dep'])
-libjsonglib = dependency('json-glib-1.0', version: '>= 1.6.0')
+libjsonglib = dependency('json-glib-1.0', version: '>= 1.4.4')
```

Comment:
this was kinda deliberate -- main is now targetting newer distros than LTS I'm afraid.

Reply:
Newer than Ubuntu _20.04_ LTS specifically.  _22.04_ should be new enough, no?

Reply:
ok
I will keep this locally while waiting for time to upgrade my distrib